docs/architecture/microservices/multi-container-microservice-net-applications/database-server-container.md

```diff
@@ -166,30 +251,30 @@ docker run --name some-redis -d redis
 
 The Redis image includes expose:6379 (the port used by Redis), so standard container linking will make it automatically available to the linked containers.
 
-In eShopOnContainers, the basket.api microservice uses a Redis cache running as a container. That basket.data container is defined as part of the multi-container docker-compose.yml file, as shown in the following example:
+In eShopOnContainers, the basket-api microservice uses a Redis cache running as a container. That basketdata container is defined as part of the multi-container docker-compose.yml file, as shown in the following example:
 
 ```yml
 #docker-compose.yml file
 #...
-  basket.data:
+  basketdata:
     image: redis
     expose:
       - "6379"
 ```
 
-This code in the docker-compose.yml defines a container named basket.data based on the redis image and publishing the port 6379 internally, meaning that it will be accessible only from other containers running within the Docker host.
+This code in the docker-compose.yml defines a container named basketdata based on the redis image and publishing the port 6379 internally, meaning that it will be accessible only from other containers running within the Docker host.
```

Comment:
```suggestion
This code in the docker-compose.yml defines a container named `basketdata` based on the redis image and publishing the port 6379 internally. This means that it will only be accessible from other containers running within the Docker host.
```